src/main/java/uk/gov/digital/ho/hocs/workflow/api/WorkflowService.java

```diff
@@ -78,6 +110,13 @@ public CreateCaseResponse createCase(String caseDataType, LocalDate dateReceived
             seedData.putAll(data);
             camundaClient.startCase(caseUUID, caseDataType, seedData);
 
+            //Create correspondent
+            if (correspondentRequest != null) {
+                UUID stageUUID = migrationCaseworkClient.getStageUUID(caseUUID);
+                UUID correspondentUUID = migrationCaseworkClient.saveCorrespondent(caseUUID, stageUUID, correspondentRequest);
```

Comment:
In the line here... we're using a service that's been written for "Migration" purposes. I think you should switch this out for a dedicated `saveCorrespondence` method. I know you would have to write it, however using this doesn't give any ambiguous use cases. 

So can I request you replace these methods and the use of the migrationCaseworkClient please with methods in the caseworkClient that call the equivalent endpoints in the caseworker service:

`/active-stage/case/{caseUUID}` and `/case/{caseUUID}/stage/{stageUUID}/correspondent`

In the latter endpoint's method, it automatically assigns the primary user so you won't need the next line.